addons/web/static/src/scss/report.scss

```diff
@@ -64,3 +64,11 @@ div#total {
         }
     }
 }
+
+/* Wkhtmltopdf doesn't handle flexbox properly */
+.col-auto{
+    // grow with content without shrinking whole pdf
+    -webkit-box-flex: 1 !important;
```

Comment:
I don't understand the comment. For me it is the exact opposite? This will now be full width and potentially be shrunk to stay on one line instead of growing with the content? I am not sure that particular comment is needed though, only a general comment on top of `col-auto` to explain why we need to override it is important. Comments to explain css rules are not :wink: 

Also:
> it is needed because of no/broken support by wkhtmltopdf

The point was that `-webkit-box-` is added automatically by our code. So writing it yourself should not have any effect at all.

Reply:
updated again, thanks for the comments and suggestions.

This is exactly the purpose actually, column content wrapping is broken so when there is too much content it will try to display everything on the same line and it will shrink the font.
With the patch the content of the column will wrap, not the columns, so when there are too many they will display packed tightly

In bootstrap.css we have defined flex as `0 0 auto` but from my tests even putting `1 0 auto` did not help :/

https://github.com/odoo/odoo/blob/4289d266320396c49c2f735d2c987627db615e75/addons/web/static/lib/bootstrap/css/bootstrap.css#L628-L632